services/resource-usage-tracker/tests/unit/with_dbs/test_api_resource_tracker__list.py

```diff
@@ -0,0 +1,135 @@
+from datetime import datetime, timezone
+from typing import Any, Final, Iterator
+from unittest import mock
+
+import faker
+import httpx
+import pytest
+import sqlalchemy as sa
+from pytest_mock import MockerFixture
+from simcore_postgres_database.models.resource_tracker import resource_tracker_container
+from starlette import status
+from yarl import URL
+
+pytest_simcore_core_services_selection = [
+    "postgres",
+]
+pytest_simcore_ops_services_selection = [
+    "adminer",
+]
+
+FAKE: Final = faker.Faker()
+
+
+@pytest.fixture
+def mocked_setup_background_task(mocker: MockerFixture):
+    mocked_background_task = mocker.patch(
+        "simcore_service_resource_usage_tracker.core.application.setup_background_task",
+        autospec=True,
+    )
+    return mocked_background_task
+
+
+@pytest.fixture
+def mocked_setup_prometheus_api_client(mocker: MockerFixture):
+    mocked_setup_prometheus_api_client = mocker.patch(
+        "simcore_service_resource_usage_tracker.core.application.setup_prometheus_api_client",
+        autospec=True,
+    )
+    return mocked_setup_prometheus_api_client
+
+
+def random_resource_tracker_container(**overrides) -> dict[str, Any]:
+    """Generates random fake data projects DATABASE table"""
+    data = dict(
+        container_id=FAKE.uuid4(),
+        image="registry.osparc.io/simcore/services/dynamic/jupyter-smash:3.0.7",
+        user_id=FAKE.pyint(),
+        project_uuid=FAKE.uuid4(),
+        product_name="osparc",
+        service_settings_reservation_nano_cpus=None,
+        service_settings_reservation_memory_bytes=None,
+        service_settings_reservation_additional_info={},
+        container_cpu_usage_seconds_total=FAKE.pyint(),
+        prometheus_created=datetime.now(tz=timezone.utc),
+        prometheus_last_scraped=datetime.now(tz=timezone.utc),
+        modified=datetime.now(tz=timezone.utc),
+        node_uuid=FAKE.uuid4(),
+        node_label=FAKE.word(),
+        instance="gpu",
+        service_settings_limit_nano_cpus=None,
+        service_settings_limit_memory_bytes=None,
+        project_name=FAKE.word(),
+        user_email=FAKE.email(),
+        service_key="simcore/services/dynamic/jupyter-smash",
+        service_version="3.0.7",
+    )
+
+    data.update(overrides)
```

Comment:
you can create a fixture that would do so:
```python
@pytest.fixture
def random_resource_tracker_container(faker: Faker) -> Callable[..., dict[str, Any]]:
    def _creator(**overrides) -> dict[str, Any]:
        data = dict(
            container_id=FAKE.uuid4(),
            image="registry.osparc.io/simcore/services/dynamic/jupyter-smash:3.0.7",
            user_id=FAKE.pyint(),
            project_uuid=FAKE.uuid4(),
            product_name="osparc",
            service_settings_reservation_nano_cpus=None,
            service_settings_reservation_memory_bytes=None,
            service_settings_reservation_additional_info={},
            container_cpu_usage_seconds_total=FAKE.pyint(),
            prometheus_created=datetime.now(tz=timezone.utc),
            prometheus_last_scraped=datetime.now(tz=timezone.utc),
            modified=datetime.now(tz=timezone.utc),
            node_uuid=FAKE.uuid4(),
            node_label=FAKE.word(),
            instance="gpu",
            service_settings_limit_nano_cpus=None,
            service_settings_limit_memory_bytes=None,
            project_name=FAKE.word(),
            user_email=FAKE.email(),
            service_key="simcore/services/dynamic/jupyter-smash",
            service_version="3.0.7",
        )

        data.update(overrides)
        return data
    return _creator
```

Reply:
i will add it in the upcoming PR today afternoon/tomorrow